docs/getting-started/2-installing.rst

```diff
@@ -24,6 +24,9 @@ full list of the available options can be found in the `Helm chart README`_.
    If your cluster does not use RBAC (Role Based Access Control), you
    will need to disable creation of RBAC resources by adding
    ``--set rbac.create=false`` to your ``helm install`` command above.
+   
+.. note::
+   If it tries to connect to localhost, instead of your actual Kubernetes cluster, check your kubectl context with ``kubectl    config current-context``. If that is correct, use ``kubectl patch deploy --namespace kube-system tiller-deploy -p  '{"spec":{"template":{"spec":{"automountServiceAccountToken":true}}}}'``
```

Comment:
I can see the value in including a note about ensuring your kubernetes context is set correctly, however I am not too sure why the `kubectl patch` step might be needed.

The default value for `automountServiceAccountToken` is already true as far as I am aware?

Reply:
@tijsg Ping on the above 😄

Reply:
@munnerz Hi James, can we add this without the "kubectl patch" part and close this one?

Reply:
@tijsg yep if you're able to update the PR to not include the patch change, I'll add lgtm 😄